src/components/Card.js

```diff
@@ -10,6 +10,17 @@ const VERTICAL_ALIGN = {
 };
 
 class Card extends PureComponent {
+  handleClick = e => {
+    e.preventDefault();
+    const { onClick, text } = this.props;
+
+    if (onClick) {
```

Comment:
Lets change this, only if onclick exists trigger `onClick` lets not execute alert